alchemiscale/models.py

```diff
@@ -31,7 +31,13 @@ def _validate_component(v, component):
         if v is not None and "-" in v:
             raise ValueError(f"'{component}' must not contain dashes ('-')")
         elif v == "*":
-            return None
+            # if we're given an asterisk, cast this to `None` instead for
+            # consistency
+            v = None
+        elif v is not None and "*" in v:
+            raise ValueError(
+                f"asterisks ('*') for glob-matching components not supported at this time"
```

Comment:
This is "glob-matching" as separate from scope-glob matching right? 

This dissallows "my*org-blah-blah" ?

Reply:
Correct! We don't want to support e.g. `"my*org-blah-blah"`, so we scream here if that is attempted.